### PR TITLE
ceph-volume: allow listing devices by OSD ID

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
## Checklist
- [x] References tracker ticket
- [x] Updates documentation if necessary
- [x] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by rishabh-d-dave** [comment]
jenkins test ceph-volume all

**Comment by rishabh-d-dave** [comment]
jenkins test ceph-volume tox

**Comment by rishabh-d-dave** [comment]
@jan--f Can you please review this PR?

File: src/ceph-volume/ceph_volume/devices/lvm/listing.py [hunk 1]
```diff
@@ -140,32 +139,28 @@ def full_report(self):
...
+        if isinstance(arg, int) or arg.isdigit():
+            lv = api.get_lvs_from_osd_id(arg)
+        elif arg[0] == '/':
+            lv = api.get_lvs_from_path(arg)
         else:
-            # vg_name/lv_name was passed
-            vg_name, lv_name = device.split('/')
-            lvs = api.get_lvs(filters={'lv_name': lv_name,
-                                       'vg_name': vg_name})
+            lv = [api.get_first_lv(filters={'lv_name': arg.split('/')[1]})]
```
Comment (guits) [inline src/ceph-volume/ceph_volume/devices/lvm/listing.py:155]: I think this might cause issue if we have multiple LVs having the same name

Comment (guits) [inline src/ceph-volume/ceph_volume/devices/lvm/listing.py:155]: @rishabh-d-dave would something like following work?
```suggestion
            lv = [api.get_first_lv(filters={'lv_name': arg.split('/')[1], 'vg_name': arg.split('/')[0]})]
```

**Comment by rishabh-d-dave** [comment]
jenkins test docs

**Comment by rishabh-d-dave** [comment]
jenkins test ceph-volume tox

**Comment by rishabh-d-dave** [comment]
jenkins test ceph-volume all

**Comment by rishabh-d-dave** [comment]
jenkins test docs

**Comment by rishabh-d-dave** [comment]
The test "centos8-filestore-mixed_type" ended with "The slave crashed or was restarted". See https://jenkins.ceph.com/job/ceph-volume-prs-batch-centos8-filestore-mixed_type/177/. Retriggering it.

**Comment by rishabh-d-dave** [comment]
jenkins test ceph-volume centos8-filestore-mixed_type

**Comment by rishabh-d-dave** [comment]
https://jenkins.ceph.com/job/ceph-volume-prs-batch-centos8-filestore-mixed_type/177/consoleFull#-159511443257ab911f-6e82-49bc-96c6-1e06cd34420e

**Comment by rishabh-d-dave** [comment]
jenkins test ceph-volume centos8-filestore-mixed_type

**Comment by rishabh-d-dave** [comment]
jenkins test ceph-volume ceph-volume-prs-batch-centos8-filestore-mixed_type

**Comment by rishabh-d-dave** [comment]
jenkins test ceph-volume batch-centos8-filestore-mixed_type

**Comment by rishabh-d-dave** [comment]
jenkins test ceph-volume centos8-filestore-mixed_type

**Comment by rishabh-d-dave** [comment]
jenkins test ceph-volume batch centos8-filestore-mixed_type

**Review by guits — APPROVED** [review]

**Pull request merged**